### ISSUE TITLE
patch introduced breaking change

### ISSUE THREAD

**Issue by gantrior** [MichaelTaylor3D/gulp-cordova-bump#17]
since v1.3.2 I am getting following error `Task bump can't support dependencies that is not an array of strings` for following usage:
```
gulp.task('bump', require('gulp-cordova-bump'));
```

please do not introduce breaking changes with minor/patch increments.. a lot of people use semantic versioning package.json like this:

~1.3.1
^1.3.1

and this shouldn't introduce breaking change...see: https://docs.npmjs.com/getting-started/semantic-versioning

> Bug fixes and other minor changes: Patch release, increment the last number, e.g. 1.0.1
> New features which don't break existing features: Minor release, increment the middle number, e.g. 1.1.0
> Changes which break backwards compatibility: Major release, increment the first number, e.g. 2.0.0

I don't know how many people are using this package, but I would recommend to revert 1.3.2 (as 1.3.3) and introduce 2.0.0 with new stuff (where the usage is probably a little different). 

Thank you for your work anyway

**Comment by Maistho** [comment]
I agree, since there was a breaking API change this should've been added as version 2. It's fine to introduce changes, but please use semver. :)
